definitions/modules/physics.yaml

```diff
@@ -36,42 +36,61 @@ master:
     var_type: f64
     interpolate: Wrap360
   -
-    type: NumSet
-    var_name: A:YOKE X POSITION
-    var_units: Position 16k
+    type: var
+    var_name: A:VELOCITY BODY X
+    var_units: Feet per second
     var_type: f64
-    event_name: AXIS_AILERONS_SET
-    interpolate: Invert
+    unreliable: true
   -
-    type: NumSet
-    var_name: A:YOKE Y POSITION
-    var_units: Position 16k
+    type: var
+    var_name: A:VELOCITY BODY Y
+    var_units: Feet per second
     var_type: f64
-    event_name: AXIS_ELEVATOR_SET
-    interpolate: Invert
+    unreliable: true
   -
-    type: NumSet
-    var_name: A:RUDDER PEDAL POSITION
-    var_units: Position 16k
+    type: var
+    var_name: A:VELOCITY BODY Z
+    var_units: Feet per second
     var_type: f64
-    event_name: AXIS_RUDDER_SET
-    interpolate: Invert
-  -
-    type: NumSet
-    var_name: A:BRAKE LEFT POSITION EX1
-    var_units: Position 32k
-    var_type: i32
-    event_name: AXIS_LEFT_BRAKE_SET
-    add_by: -16384
     unreliable: true
+
+init:
```

Comment:
same as above

Reply:
Added because of trouble syncing location to host upon connection. Sync (in v2.3) only appeared to work upon movement and not on initial connect. This is to avoid having to "slew" to correct.

Reply:
If the definition is under shared, it will be synced on initial connection regardless. Init specifies that the definition will only be synced *once* and never again.